uiplib/scrape/scrape.py

```diff
@@ -19,7 +22,10 @@ def make_json(url):
     """
     makes a dictionary out of a json file.
     """
-    response = requests.get(url, headers={'User-agent': 'UIP'})
+    response = requests.get(
+        url,
+        headers={'User-agent': 'UIP'},
+        proxies=getproxies())
```

Comment:
Can you add a comment explaining what this argument and the function called does?